Annotato/Annotato/Views/Document/Edit/DocumentEditViewController.swift

```diff
@@ -11,7 +12,12 @@ class DocumentEditViewController: UIViewController {
 
     func initializeSubviews() {
         initializeToolbar()
-        initializeDocumentView()
+
+        if let currentDocumentViewModel = currentDocumentViewModel {
+            initializeDocumentView(documentViewModel: currentDocumentViewModel)
+        } else {
+            return
+        }
```

Comment:
Maybe we can use a guard clause here, since the happy path should be having a view model. And we want to guard against not having one.
```suggestion
        guard currentDocumentViewModel != nil else {
            return // or present an error alert (this is better), it can be easily done by conforming to AlertPresentable
        }
        
        initializeDocumentView(documentViewModel: currentDocumentViewModel)
```

Reply:
Ok I changed it to present an alert
I'll resolve this